File: app/src/main/java/com/pdamkotasmg/happywork/api/server/ApiConfig.java
package com.pdamkotasmg.happywork.api.server;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class ApiConfig {

    public static ApiService getApiService(){
        Retrofit retrofit = new Retrofit.Builder()
                .baseUrl("https://app.pdamkotasmg.co.id/api-gw-dev/portal-pegawai/api/")
                .addConverterFactory(GsonConverterFactory.create())
                .build();

        return retrofit.create(ApiService.class);
    }
}

File: app/src/main/java/com/pdamkotasmg/happywork/api/server/ApiConfig.kt
package com.pdamkotasmg.happywork.api.server

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object ApiConfig {
    fun getApiService(): ApiService? {
        val retrofit = Retrofit.Builder()
                .baseUrl("https://app.pdamkotasmg.co.id/api-gw-dev/portal-pegawai/api/")
                .addConverterFactory(GsonConverterFactory.create())
                .build()
        return retrofit.create(ApiService::class.java)
    }
}